Add arrow and offsets props to the Popover (#6)
vtex-apps/popover-layout: docs/README.md

| `placement` | `Placement` | Describes the preferred placement of the popover. If there is no space to put the popover in the placement that you choose it will fit in a fallback position | `'bottom`' |
| `scrollBehavior` | `ScrollBehavior` | How the popover should behave if the user tries to scroll the page | `'default'` |
| `backdrop` | `BackdropOptions` | If it should display a backdrop overlay | `'none'` |

> | `arrow`  | `boolean` | If it should display an arrow pointing to the `popover-trigger` element | `false` |

Reviewer: Maybe `arrow` is a too generic name. How about `showArrow`, or `popoverArrow`?

| `'visible'` | Displays a backdrop overlay which blocks the click on other items of the page, if you click on the overlay it will close the popover |
| `'none'` | Doesn't display any backdrop overlay and if you click any item in the page it will close the popover and click the element |

### Offsets

> | Property | Type | Description |
> | --- | --- | --- |
> | `skidding` | `number` | Displaces the `popover-layout` along the `popover-trigger`. |
> | `distance` | `number` | Displaces the `popover-layout` away from, or toward, the `popover-trigger` in the direction of its placement. A positive number displaces it further away, while a negative number lets it overlap the `popover-trigger`. |

Reviewer 2: ```suggestion
| Value | Description |
| --- |  --- |
| `skidding` | Displaces the `popover-layout` along the `popover-trigger`. |
| `distance` | Displaces the `popover-layout` away from, or toward, the `popover-trigger` in the direction of its placement. A positive number displaces it further away, while a negative number lets it overlap the `popover-trigger`. |
```

> ### PopoverTrigger

Reviewer 2: ```suggestion
## PopoverTrigger
```

Reviewer 2: Do the same thing to ### Popover

| Prop name | Type | Description | Default value |
| --- | --- | --- | --- |

> | trigger | `TriggerMode` | What kind of user action must happen to open the `popover-layout` | `'click'` |

Reviewer 2: ```suggestion
| trigger | `TriggerMode` | What kind of user action must happen to open the `popover-layout`.<br/> - `click`<br/>- `hover` | `'click'` |
```

Author: I think we could talk with @guerreirobeatriz about how to do this and change the structure of the documentation in general

### TriggerMode

| Value | Description |
| --- | --- |
| `'click'` | Open when you click the `popover-trigger` |

> | `'hover'` | Open when you hover the `popover-trigger` | 

Reviewer 2: Remove
